applications/registration-react/src/services/auth/auth-service.js

```diff
@@ -13,49 +12,62 @@ const PERMISSION_ADMIN = 'PERMISSION_ADMIN';
 const RESOURCE_ORGANIZATION = 'organization';
 const ROLE_ADMIN = 'admin';
 
+const redirectPath = '/auth_response';
+const loginPath = '/login';
+const redirectUri = `${location.origin}${redirectPath}`;
+
 function toPromise(keycloakPromise) {
   return new Promise((resolve, reject) =>
     keycloakPromise.success(resolve).error(reject)
   );
 }
 
-function setOnAuthSuccess() {
-  const handler = () => {
-    storeTokens({ token: kc.token, refreshToken: kc.refreshToken });
-  };
-
-  kc.onAuthRefreshSuccess = handler;
-  kc.onAuthSuccess = handler;
-}
-
 function initializeKeycloak() {
-  const { token, refreshToken } = loadTokens();
   const initOptions = {
     onLoad: 'check-sso',
-    redirectUri: `${location.origin}/auth_response`,
-    token,
-    refreshToken
+    redirectUri
   };
 
   return toPromise(kc.init(initOptions));
 }
 
 export const isAuthenticated = () => kc && kc.authenticated;
 
+function storePotentialDirectNavigation() {
+  const { redirectLocation } = getLoginState();
+
+  // is in redirect
+  if (redirectLocation) {
+    return;
+  }
```

Comment:
This is not very clear at first glance, but I think I get the point

Reply:
trenger kommentar da, takk!